styles.css

```
@@ -0,0 +1,1985 @@
+*,
+*::before,
+*::after {
+    box-sizing: border-box;
+}
+
+.logo {
+    background-color: transparent;
+    height: 239px;
+    width: 240px;
+    margin-left: 0px;
+    margin-right: 0px;
+    padding-bottom: 0;
+    position: relative;
+    left: 65px;
+    right: 48px;
+}
+
+h1 {
+    color: black;
+    font-family: 'Teko', sans-serif;
+    font-size: 48px;
+    margin-bottom: 30px;
+    margin-top: 20px;
+    max-width: 375px;
+    line-height: 45px;
+    text-align: center;
+}
+
+h2 {
+    color: #163346;
+    font-family: 'Teko', sans-serif;
+    font-size: 26px;
+    font-weight: 400;
+    height: 60px;
+    margin-bottom: 57px;
+    margin-top: 20px;
+    max-width: 375px;
+    text-align: center;
+}
+
+h3 {
+    border-bottom: 4px solid #163346;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    height: 50px;
+    margin-bottom: 20px;
+    margin-top: 0px;
+    letter-spacing: 1px;
+    width: 140px;
+    text-align: left;
+}
+
+.character {
+    border-bottom: 4px solid #163346;
+    color: #eaecf0;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    font-weight: 300px;
```

`font-weight` values can be _either_ keywords (like `bold`) or values of 100 (like `100` or `300`), but the latter do _not_ use units of measurement.
```suggestion
    font-weight: 300;
```